content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -314,7 +314,7 @@ kubectl delete deployments,svc my-nginx; kubectl create -f ./nginx-secure-app.ya
 At this point you can reach the nginx server from any node.
 
 ```shell
-kubectl get pods -o yaml | grep -i podip
+kubectl get pods -l run=my-nginx -o yaml | grep -i podip
     podIP: 10.244.3.5
 node $ curl -k https://10.244.3.5
```

Comment:
This line should be changed as well?

Reply:
Hey @tengqm I'm getting confused with this issue, isn't this the same command which is to be changed according to [this issue](https://github.com/kubernetes/website/issues/34388)

Reply:
What I was suggesting is to move the `curl` command out of the Shell snippet because it is a different operation in a separate context.

Reply:
@chetak123 You're right this PR does change the same command the issue talks about but this PR won't close #34388 this PR just changes the part where we

get pods -> get pods with label run=my-nginx

#34388 though is not about this

Reply:
Ah, yes thanks for pointing it out I will do that

Reply:
@chetak123 You're right this PR does change the same command the issue talks about but this PR won't close https://github.com/kubernetes/website/issues/34388 this PR just changes the part where we

get pods -> get pods with label run=my-nginx

https://github.com/kubernetes/website/issues/34388 though is not about this

Reply:
@Rishit-dagli you should:

- revise this PR in line with https://github.com/kubernetes/website/pull/34399/files#r901066924
- Change the PR title and description to reflect the broader scope

Reply:
Thanks @sftim 

- Done via https://github.com/kubernetes/website/pull/34399/commits/5242784e76ebcdb64acff5d692e0aa8bdf354ea1
- Done